README.md

```diff
@@ -6,6 +6,13 @@
 $ bower install gaia-components/gaia-switch
 ```
 
+Then include folowing files in HTML
+
+```html
+<script src="bower_components/drag/drag.js"></script>
+<script src="bower_components/gaia-switch/gaia-switch.js"></script>
+```
```

Comment:
If using proper module system (AMD or Common JS) we aim to make usable as simple as:

```js
require('gaia-switch');
```

So user doesn't need to know about sub-dependencies.